### ISSUE TITLE
Export of FBX object fails - no animation data?

### ISSUE THREAD

**Issue by jgwinner** [Kupoman/blendergltf#104]
Hi. I'm loading this object:

[https://www.sharecg.com/v/76031/browse/5/3D-Model/Free-Hedges](https://www.sharecg.com/v/76031/browse/5/3D-Model/Free-Hedges)

and I get the following errors when I export:

```
Traceback (most recent call last):
  File "C:\Users\John\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\ad
dons\blendergltf-master\__init__.py", line 370, in execute
    gltf = export_gltf(data, settings)
  File "C:\Users\John\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\ad
dons\blendergltf-master\blendergltf.py", line 1592, in export_gltf
    state['output']['animations'] = export_animations(state, scene_delta.get('ac
tions', []))
  File "C:\Users\John\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\ad
dons\blendergltf-master\blendergltf.py", line 1442, in export_animations
    export_active(regular_objects)
  File "C:\Users\John\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\ad
dons\blendergltf-master\blendergltf.py", line 1425, in export_active
    if obj.animation_data.action:
AttributeError: 'NoneType' object has no attribute 'action'

location: <unknown location>:-1

location: <unknown location>:-1
```

#### Steps to reproduce:

Download referenced model from sharecg

Import via FBX import into Blender 2.79
Export via gLTF exporter.

I attached a screenshot of the options.

### Question: 

In the export options, isn't there a way to turn off exporting animations? I tried "Active Only" but that didn't help the error.

[Free Hedges.zip](https://github.com/Kupoman/blendergltf/files/1367768/Free.Hedges.zip)
![gltfbugoptions](https://user-images.githubusercontent.com/10425170/31328517-72a612b4-ac8a-11e7-81e7-94b3b02fdf14.png)

**Comment by jgwinner** [comment]
I did find if I created a keying for location for both the object and the camera (Camera seems to be the needed one), and added one keyframe, it seemed to work.

== John ==

**Comment by jgwinner** [comment]
I would call this more of a work around than 'there's no problem'.

I'm using the other gLTF importer until this is fixed.

**Comment by Kupoman** [comment]
This bug should be fixed by 8fbb66aaa6f0f3b167ba7ef940218a06a9e0e882, and was automatically closed by that commit.

**Comment by jgwinner** [comment]
Ah, got it - thanks!

== John ==
